### DISCUSSION TITLE
「开源之史」系列之四/六：自由软件运动的源头——黑客文化简史 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#78] [Announcements]
# 「开源之史」系列之四/六：自由软件运动的源头——黑客文化简史 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/04-02-hacker-culture/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

黑客行话 就象那个惊天而出的 全球概览(WEC,Whole Earth Catalog);
在互联网构建时期, 
所有互联网内容/黑客文化/网站概要, 还是能用一个目录, 一个杂志来承载的;

不过, 互联网的指数级增长, 很快就超越了传统形式可以承载的上限,
所以, Jargon File 停止增补了, 就象 Yahoo! 的目录服务, 
因为投入产出已经不成比例,
或是说, 多黑话, 要嘛已经弃用, 要嘛已经变成日常厘语, 不值得专门立目录来索引,
也就是说, 无人使用了...

这其实意味着几个方向上的变化:

+ Hacker 已经文化极大丰富, 不是一个 Jargon File 可以涵盖的了
+ Hacker 生态已经极大友好, 不是必须 Jargon File 学习才能进入的了
+ Hacker 工具极大丰富, 并不是只能在 Jargon File 上可以增补
+ ...

对比中国 CPyUG 也一样, 开始并没有什么专业门户, 大家都在自发的分享资料,
然后通过 mailman 聚集起来, 共同发布维护一个 moinmoin 维基,
接下来, 越来越多方便的资料发布/分享形式出现,
开源社区基础设施:

+ 代码仓库
+ 邮件列表
+ 知识仓库
+ 问题追踪

也分别被替代或是融合, 或是不再必要...

文化也一样...

> 永远不变的只有变化本身

**Reply by lijiangsheng1** [reply]
下一节谈全球概览～

**Reply by ZoomQuiet** [reply]
以及反战, 雅痞文化, make love no war ...
这都是 自由软件运动的精神成分
所以有 [黑客伦理与信息时代精神 (The Hacker Ethic and the Spirit of the Information Age)](https://book.douban.com/subject/1071093/)
